### ISSUE TITLE
Regarding recipes: is only "true"/"false" allowed in the available field?

### ISSUE THREAD

**Issue by lmintmate** [emacs-themes/emacs-themes-site#10]
I noticed that the site mentions such themes as blue-mood as not included/available in a package, which isn't the case, as e.g. I have blue-mood installed as part of the color-theme-modern package, and the same goes with every other theme from the replace-colorthemes repository. I looked at the recipes to see if I could write something like "Available as part of the color-theme-modern package", but the only two instances I saw in the available category are "true" and "false", as in they either are available as packages by themselves or not. The problem with that dichotomy though is that people browsing the site can be discouraged when they see that a theme the appearance of which they liked supposedly doesn't have a package, which increases the perceived difficulty curve of its installation, and discourages them from even trying less popular themes.
In short, I wonder whether in these cases, the "false" could be replaced by something like "true, as part of the color-theme-modern package" or (in case "true" adds, as part of the recipe, a link to the standalone package and assumes its name based on the name of the theme) "false -not available as standalone- but available as part of the color-theme-modern package".
Thanks in advance for understanding.

**Comment by caisah** [comment]
Hi, 

Thanks for reaching out at me. Your point is valid and seems like a very nice improvement. I'll find a way to address this but I won't promise it will be right away. It will take a few days. When I'm done I'll let you know here and I'll document the changes.
